src/App.ts

```diff
@@ -90,4 +85,56 @@ export default class App {
 			console.log(`Listening on http://${this.host}:${this.port}/`);
 		});
 	}
+
+	/**
+	 * Mount the root route.
+	 */
+	private initRoot() {
+		// Set up a root route
+		this.app.get('/', (_req: Request, res: Response) =>
+			res.send({
+				docs: 'https://paritytech.github.io/substrate-api-sidecar/dist',
```

Comment:
This gives me a 404, is it just not published yet?

Reply:
Are you running this specific branch of Sidecar?